tooling/cli/src/utils/init-cli.ts

```diff
@@ -2,7 +2,7 @@ import checkNode from "cli-check-node"
 import welcome from "cli-welcome"
 import updateNotifier from "update-notifier"
 import unhandledError from "cli-handle-unhandled"
-import * as pkgJSON from "../../package.json"
```

Comment:
I had to update this because TypeScript was complaining that the `description` and `version` fields weren't exported. Since `package.json` is a single object, there's not anything to collect with `*` anyway.